README.md

```diff
@@ -33,19 +33,19 @@ A sample app for testing purposes is automatically built after commits in the re
 ### Screenshots of a sample Android app
 
 <p align="center">
-<img src="docs/images/app-drawer.png" height = "480" width="270"> <img src="docs/images/app-tracks.png" height = "480" width="270"> <img src="docs/images/app-schedule.png" height = "480" width="270">
+<img src="docs/_static/images/app-drawer.png" height = "480" width="270"> <img src="docs/_static/images/app-tracks.png" height = "480" width="270"> <img src="docs/_static/images/app-schedule.png" height = "480" width="270">
```

Comment:
3 should not be a issue. It only happens when theme is alabaster. Keep 3 images in a row as the images are logically arranged according to that. Can be changed later on. Other than that look good.